scripts/setup-branch-protection.sh

```
@@ -0,0 +1,196 @@
+}
+
+# Check if gh CLI is installed and authenticated
+check_prereqs() {
+    print_status "Checking prerequisites..."
+
+    if ! command -v gh &> /dev/null; then
+        print_error "GitHub CLI (gh) is not installed. Please install it first."
+        exit 1
+    fi
+
+    if ! GH_PAGER=cat gh auth status &> /dev/null; then
```

Shellcheck (0.10.0)

[warning] Use var=$(command) to assign output (or quote to assign string).

(SC2209)